tests/ballerina-spec-conformance-tests/src/test/java/org/ballerinalang/test/TestRunnerUtils.java

```diff
@@ -111,17 +111,18 @@ private static void readTestFile(File testFile, String tempDir, String fileName,
             Map<String, String> headersOfTestCase = readHeaders(line, buffReader);
             String kindOfTestCase = validateKindOfTest(headersOfTestCase.get(TEST_CASE));
 
+            String labels = headersOfTestCase.get(LABELS);
+            String[] testCaseLabels = labels.split("\\s*,\\s*");
+
             //TODO: if kind of testcase is other, then need to skip creating the bal file
-            boolean isSkippedTestCase = isSkippedTestCase(selectedLabels, headersOfTestCase.get(LABELS));
```

Comment:
Now the `selectedLabels` argument passing to this method is redundant.

Reply:
Sorry, I didn't get you. This method is no longer used.

Reply:
sorry. What I meant by the method is `readTestFile()`. `selectedLabels` argument passing to that has no usage within.

Reply:
Ahh.. yes! missed that. will update. :)